documentation/asciidoc/accessories/camera/libcamera_apps_getting_started.adoc

```diff
@@ -0,0 +1,37 @@
+=== Getting Started
+
+==== Preparing your Pi for the first time
+
+The 5 basic `libcamera-apps` should be installed as part of the Raspberry Pi operating system. Before a sensor can be used, however, the user must check that the correct Linux kernel driver is being loaded. This will involve including a `dtoverlay` line in the `/boot/config.txt` file, according to the sensor being used.
+
+|===
+| Camera Module | In `/boot/config.txt`
+
+| V1 camera (OV5647)
+| `dtoverlay=ov5647`
+
+| V2 camera (IMX219)
+| `dtoverlay=imx219`
+
+| HQ camera (IMX477)
+| `dtoverlay=imx477`
+
+| IMX290
+| `dtoverlay=imx290,clock-frequency=74250000` or `dtoverlay=imx290,clock-frequency=37125000`
+
+| IMX327
+| `dtoverlay=imx290,clock-frequency=74250000` or `dtoverlay=imx290,clock-frequency=37125000`
```

Comment:
Was this a cut'n'paste typo, or does the IMX327 indeed use the exact same dtoverlay as IMX290?